core/src/serve_repair.rs

```diff
@@ -65,6 +67,7 @@ pub enum ShredRepairType {
     Orphan(Slot),
     HighestShred(Slot, u64),
     Shred(Slot, u64),
+    Coding(Slot, u64),
```

Comment:
Any changes to protocol messages should go in first as separate commit. Once that is merged in, backported, released and the cluster has upgraded, then the rest of changes using those new protocol messages can be merged into master.
Otherwise a node running the new code will send messages that other nodes cannot deserialize, and it breaks backward compatibility.